logger/logger.config

```diff
@@ -7,7 +7,7 @@
 // Do not leave any setting blank.
 
 // Output logs to terminal (stdout)? (Yes/No)
-LOG_TO_STDOUT: YES
+LOG_TO_STDOUT: NO
```

Comment:
That will make `net_handler_cli` output double the logs (because both standard output and the network are producing the same logs). Plus, this is the `logger.config` that will be used in production, and we shouldn't be printing the logs to standard out in production, right?

Reply:
Ok makes sense.